Reference/Searching/Examine/transforming-index-values.md

```diff
@@ -0,0 +1,120 @@
+---
+versionFrom: 8.0.0
+---
+
+# TransformingIndexValues
```

Comment:
The file name and main headline of this page should be changed to examine-events (see above comment)